mmv1/README.md

```diff
@@ -25,6 +25,101 @@ We're glad to accept contributions to MMv1-generated resources. Tutorials and gu
+for IAM support can be inferred based on the preexisting YAML specification file.
+
+To add support for IAM resources based on an existing resource, add an
+`iam_policy` block to the resource's definition in `api.yaml`, such as the
+following:
+
+```yaml
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
+      method_name_separator: ':'
+      fetch_iam_policy_verb: :POST      
+      parent_resource_attribute: 'registry'
+      import_format: ["projects/{{project}}/locations/{{location}}/registries/{{name}}", "{{name}}"]         
+```
+
+The specification values can be determined based on a mixture of the resource
+specification and the cloud.google.com `setIamPolicy`/`getIamPolicy` REST
+documentation, such as
+[this page](https://cloud.google.com/iot/docs/reference/cloudiot/rest/v1/projects.locations.registries/setIamPolicy)
+for Cloud IOT Registries.
+
+`parent_resource_attribute` - (Required) determines the field name of the parent
+resource reference in the IAM resources. Generally, this should be the singular
+form of the parent resource kind in camel case, i.e. `registries` -> `registry`
+or `backendServices` -> `backend_service`.
+
+`method_name_separator` - (Required) should be set to the character preceding
+`setIamPolicy` in the "HTTP Request" section on the resource's `setIamPolicy`
+page. This is almost always `:` for APIs other than Google Compute Engine (GCE),
+MMv1's `compute` product.
+
+`fetch_iam_policy_verb` - (Required) should be set to the HTTP verb listed in
+the "HTTP Request" section on the resource's `getIamPolicy` page. This is
+generally `POST` but is occasionally `GET`. Note: This is specified as a Ruby
+symbol, prefixed with a `:`. For example, for `GET`, you would specify `:GET`.
+
+`import_format` - (Optional) A list of templated strings used to determine the
+Terraform import format. If the resource has a custom `import_format` or
+`id_format` defined in `terraform.yaml`, this must be supplied.
+    * If an `import_format` is set on the parent resource use that set of values exactly, substituting `parent_resource_attribute` for the field name of the **final** templated value.
+    * If an `id_format` is set on the parent resource use that as the first entry (substituting the final templated value, as with `import_format`) and define a second format with **only** the templated values, `/`-separated. For example, `projects/{{project}}/locations/{{region}}/myResources/{{name}}` -> `["projects/{{project}}/locations/{{region}}/myResources/{{myResource}}", "{{project}}/{{region}}/{{myResource}}"]`.
+        * Optionally, you may provide a version of the shortened format that excludes entries called `{{project}}`, `{{region}}`, and `{{zone}}`. For example, given `{{project}}/{{region}}/{{myResource}}/{{entry}}`, `{{myResource}}/{{entry}}` is a valid format. When a user specifies this format, the provider's default values for `project`/`region`/`zone` will be used.
```

Comment:
Looks like the format here is a bit off. Maybe remove the first indent in the list?